models/repo.go

```diff
@@ -729,26 +729,44 @@ func (repo *Repository) getReviewersPublic(e Engine, doerID, posterID int64) (_
 	return users, nil
 }
 
-func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) (users []*User, err error) {
-	if err = repo.getOwner(e); err != nil {
-		return nil, err
+func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) ([]*User, []*Team, error) {
+	if err := repo.getOwner(e); err != nil {
+		return nil, nil, err
 	}
 
+	var (
+		users []*User
+		err   error
+	)
+
 	if repo.IsPrivate ||
 		(repo.Owner.IsOrganization() && repo.Owner.Visibility == api.VisibleTypePrivate) {
 		users, err = repo.getReviewersPrivate(x, doerID, posterID)
 	} else {
 		users, err = repo.getReviewersPublic(x, doerID, posterID)
 	}
-	return
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if repo.Owner.IsOrganization() {
+		teams, err := GetTeamsWithAccessToRepo(repo.OwnerID, repo.ID, AccessModeRead)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		return users, teams, nil
+	}
```

Comment:
What's the relevance to getReviewers here? What's this doing in here?

Reply:
Hmm, I'd like to use one function to get them, May be should have another one, or rename this function.  Thank you for your reminding.